docs/installing.rst

```diff
@@ -56,14 +56,43 @@ Two virtualization providers have been tested with the ADB.
       $ mv virtualbox.repo /etc/yum.repos.d/
       $ yum install VirtualBox-4.3
       $ sudo /etc/init.d/vboxdrv setup
-    
-  * libvirt is shipped in CentOS and the preferred installation uses the distribution packages:
-    
-    ::
-    
-      $ yum install qemu-kvm qemu-img virt-manager libvirt libvirt-python libvirt-client virt-install virt-viewer
-      $ systemctl start libvirtd
-      $ systemctl enable libvirtd
+
+  * libvirt-provider is not available in CentOS core however it is available through Fedora Copr and `Software Collections <http://softwarecollections.org>`_.
+
+.. _vagrantRepo:
+
+::
+
+      $ cat > /etc/yum.repos.d/vagrant.repo <<- EOM
+        [jstribny-vagrant1]
+        name=Copr repo for vagrant1 owned by jstribny
+        baseurl=https://copr-be.cloud.fedoraproject.org/results/jstribny/vagrant1/epel-7-x86_64/
+        gpgcheck=1
+        gpgkey=https://copr-be.cloud.fedoraproject.org/results/jstribny/vagrant1/pubkey.gpg
+        enabled=1
+
+        [ruby200-copr]
+        name=ruby200-copr
+        baseurl=http://copr-be.cloud.fedoraproject.org/results/rhscl/ruby200-el7/epel-7-x86_64/
+        enabled=1
+        gpgcheck=0
+
+        [ror40-copr]
+        name=ror40-copr
+        baseurl=http://copr-be.cloud.fedoraproject.org/results/rhscl/ror40-el7/epel-7-x86_64/
+        enabled=1
+        gpgcheck=0
+
+      EOM
+
```

Comment:
These changes are odd because you are suggesting that a vagrant Fedora COPR be used to install software that seems to have been available outside of that COPR.  I have not had a chance to verify this instruction set, but will do so ASAP.